README.md

```diff
@@ -1,121 +1,5 @@
-<a name="readme-top"></a>
+# FCC Test Suite Template
 
-<div align="center">
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-  <br/>
-
-  <h3><b>Microverse Trials </b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 Microverse Trials <a name="about-project"></a>
-
-**Microverse Trials** Is a microverse trials free code camp project.
-
```

Comment:
- [ ] please add a more decriptive readme about your project.This  helps other users understand what your project is about.